components/progress-bar/src/progress-bar-module.ts

```diff
@@ -15,13 +15,19 @@
  */
 
 import { NgModule } from '@angular/core';
+import { DtIndicatorModule } from '@dynatrace/barista-components/core';
 
 import { DtProgressBar } from './progress-bar';
 import { DtProgressBarCount } from './progress-bar-count';
 import { DtProgressBarDescription } from './progress-bar-description';
 
 @NgModule({
-  exports: [DtProgressBar, DtProgressBarDescription, DtProgressBarCount],
+  exports: [
+    DtProgressBar,
+    DtProgressBarDescription,
+    DtProgressBarCount,
+    DtIndicatorModule,
```

Comment:
Please add breaking change notation, that the DtIndicatorModule will be it's own package in version 6.0.0.
See #210